rewards-configuration.default.yml

```diff
@@ -0,0 +1,31 @@
+content-evaluator:
+  enabled: true
+user-extractor:
+  redeem-bounty: true
+data-purge:
+  enabled: true
+#disabled: true
+formatting-evaluator:
+  enabled: true
+  scores:
+    br: 1
+    code: 1
+    p: 0.1
+    em: 0
+    img: 0
+    strong: 0
```

Comment:
Strong and italics seem to be recognized as the same item in my previous testing with the mdast library. 

The reason why this is 0 by default is because the only time we normally see italics is when a new issue is automatically generated from a comment and GitHub adds the context at the end like "written by X at Y issue"

I never seen contributors use bold either for comments or issue specifications.

Reply:
I tried to copy as close as I could to the current version. Shall we put it back to 1?

Reply:
I was just providing you context for why they are set as 0. Its fine.